test/test_nn.py

```diff
@@ -1180,14 +1180,27 @@ def compare_scaling(grads):
             for p, g in zip(l.parameters(), grads):
                 p.grad.data.copy_(g)
             norm_before = compute_norm(norm_type)
-            norm = clip_grad_norm(l.parameters(), max_norm, norm_type=norm_type)
+            norm = clip_grad_norm_(l.parameters(), max_norm, norm_type=norm_type)
             norm_after = compute_norm(norm_type)
             self.assertEqual(norm, norm_before)
             self.assertEqual(norm_before, norm_after)
             self.assertLessEqual(norm_after, max_norm)
             scale = compare_scaling(grads)
             self.assertEqual(scale, 1)
 
+    def test_clip_grad_value(self):
+        l = nn.Linear(10, 10)
+        clip_value = 2.5
+
+        grads = torch.arange(-50, 50).view(10, 10).div(5), torch.ones(10).mul(2)
+        for p, g in zip(l.parameters(), grads):
+            p._grad = Variable(g.clone().view_as(p.data))
```

Comment:
Please don't use `Variable`, it's deprecated now. `g.clone().view_as(p.data)` should be just fine

Reply:
Also, it would be good to test a case when one of the parameters has `None` grad (e.g. ignore setting the grad of bias)

Reply:
Thanks @apaszke! If I don't cast to `Variable`, I get a running error: `expected Variable or None`

Reply:
Can you please paste the full error? Are you running this code on `master`?